src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -150,6 +150,10 @@ pub struct ThinPool {
     id_gen: ThinDevIdPool,
     filesystems: Table<StratFilesystem>,
     mdv: MetadataVol,
+    /// The single DM device that the backstore presents as its upper-most
+    /// layer. All DM components obtain their storage from this layer.
+    /// The device will change if the backstore adds or removes a cache.
+    backstore_device: Device,
```

Comment:
This is the forward-looking part. We would not bother to do this if we didn't expect ot deliberately change the device at some point.